src/aboutPage/AboutPage.component.js

```diff
@@ -4,6 +4,10 @@ import PropTypes from 'prop-types';
 import AboutSection from './AboutSection.component';
 
 const parseDateFromUTCString = (utcString, d2) => {
+    if (!utcString) {
```

Comment:
Could this ever be a string with an invalid date?  Maybe we should `try`/`catch` in addition or instead?

Reply:
I was considering that option but I actually think the current implementation is better. As I see it, all we need to cater for are these two scenarios:
- We receive a valid date string for server-events that have in fact taken place
- We receive `undefined` or perhaps an `''` for things that have not happened (yet)

I'd rather not wrap everything in a try/catch because that might just mask other things going wrong on the server. I.e. suppose we would get an invalid date value. If we used a try/catch, all the fields would just render "None", which would indicate to the user that "the server event hasn't happened". But this isn't what the actual problem is. So imo this would just lead to a lot of potential confusion...

Reply:
Yeah I had the same thought... I worry, though, that if something "goes wrong on the server" and we get a defined but invalid value, the entire app crashes (as it does now) and that's a MUCH worse failure mode than incorrectly showing `None`.  

We could have a separate `Unknown` or `Invalid` display message when an error is caught if we wanted to differentiate without blowing up the whole app - if we don't break things than this label shouldn't ever show, so this might be a superfluous translation key, but I think that's a fair cost for a safer failure mode?

Reply:
After discussing on Slack we decided to go with the try/catch and to implement to distinct strings:
- An `undefined` UTC string is valid and has a meaning: the even hasn't taken place. In the UI we can recognise this by the string "Never".
- An _invalid_ UTC string is a different case. Something is actually wrong here and to signify that we show the text "Invalid date"